src/safir/http.py

```diff
@@ -0,0 +1,48 @@
+"""Utilities for using the aiohttp client in Safir-based apps.
+"""
+
+__all__ = ["init_http_session"]
+
+from typing import TYPE_CHECKING
+
+from aiohttp import ClientSession
+
+if TYPE_CHECKING:
+    from aiohttp.web import Application
+
+
+async def init_http_session(app: "Application"):
```

Comment:
Add `from __future__ import annotations` to the top of the file and then you can stop quoting the types that are only imported under `if TYPE_CHECKING`.